activities/EbookReader.activity/icons/lines.svg

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Not a very intuitive icon

Reply:
I have created a new icon. Please review.